soccer_ipm/soccer_ipm/utils.py

```diff
@@ -0,0 +1,54 @@
+from builtin_interfaces.msg import Time
+from ipm_interfaces.msg import PlaneStamped
+from sensor_msgs.msg import CameraInfo
+from vision_msgs.msg import BoundingBox2D, Point2D
+
+
+def create_field_plane(
```

Comment:
I think calling this plane the "field_plane" is a bit confusing when there is a height offset. 
May I suggest an alternative, call it the "horizontal_plane" with a ``z: float = 0.0`` parameter?